snappy/snapp_snapfs_test.go

```diff
@@ -292,7 +250,7 @@ func (s *SquashfsTestSuite) TestInstallKernelSnapUnpacksKernel(c *C) {
 		OfficialName: "ubuntu-kernel",
 		Revision:     snap.R(42),
 	}
-	_, err := (&Overlord{}).InstallWithSideInfo(snapPkg, si, 0, &MockProgressMeter{})
+	_, err := (&Overlord{}).InstallWithSideInfo(snapPkg, si, LegacyInhibitHooks, &MockProgressMeter{})
```

Comment:
@chipaca this branch hollows out the Overlord.Installs such that they don't do a full activation anymore (anyway it didn't make sense), and even if we need to resurrect u-d-f is passing InhbitHooks.

 As usual just trying to keep the tests going until they are ported
 while still emptying snappy progressively.